src/pages/PlainSuomiFiProfilePage.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PageContent from '../components/PageContent';
+import ClientGetUser from '../components/ClientGetUser';
+import LoginInfo from '../components/LoginInfo';
+import AuthenticatingInfo from '../components/AuthenticatingInfo';
+import WithAuth from '../client/WithAuth';
+
+const PlainSuomiFiProfilePage = (): React.ReactElement => (
```

Comment:
It could be confusing to call this page "profile", as it's not the "real" profile, just `userinfo`. And `userinfo` could actually be included to the "full profile" side too, perhaps as a separate tab.

Reply:
I'd rather match filenames of the pages with the names they have in the navigation and the path of the page. 

> And userinfo could actually be included to the "full profile" side too, perhaps as a separate tab.
Doesn't the token page show all data Tunnistamo would return in getUser?

Reply:
Perhaps the name should be changed in the navigation and path too...

Reply:
Current version has been approved. I'll open a new discussion for this.

Reply:
User Info was approved